modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -380,10 +390,27 @@ public function deleteReservation(AlephPatron $patron, AlephRequest
    *    The Aleph patron ID.
    *
    * @return \SimpleXMLElement
-   * @throws \RuntimeException
+   * @throws AlephClientException
    */
   public function getPatronBlocks($bor_id) {
     return $this->requestRest('GET', 'patron/' . $bor_id . '/patronStatus/blocks');
   }
 
+  /**
+   * @param \Drupal\aleph\Aleph\Entity\AlephPatron $patron
+   *    The Aleph patron.
+   * @param \Drupal\aleph\Aleph\Entity\AlephMaterial $material
+   *    The Aleph material.
+   *
+   * @return \SimpleXMLElement[]
+   * @throws AlephClientException
+   */
+  public function getHoldingGroups(AlephPatron $patron, AlephMaterial $material) {
```

Comment:
Please add a comment about what this function does.

The concept of Holding Groups is unique to Aleph.